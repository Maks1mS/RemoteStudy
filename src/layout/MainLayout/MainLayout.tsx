import styled from '@emotion/styled'

const Layout = styled.div`
  display: grid;
  height: 100%;
  grid-template-areas: "header" "main";
  grid-template-rows:  50px 1fr;
`

const Header = styled.header`
  grid-area: "header";
  height: 50px;
  background: #212529;
  color: #fff;
  display: flex;
  flex-direction: row;
`

const Main = styled.main`
  grid-area: "main";
  display: grid;
  grid-template-columns: 1fr; 
`

const MainLayout: React.FC = ({ children }) => {
  return <Layout>
    <Header>

    </Header>
    <Main>
      {children}
    </Main>
  </Layout>
}

export default MainLayout
