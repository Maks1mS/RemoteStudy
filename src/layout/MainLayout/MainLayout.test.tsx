import { render } from '@testing-library/react'
import MainLayout from './MainLayout'

test('Renders', () => {
  const { container } = render(<MainLayout>Test</MainLayout>)
  expect(container).toMatchSnapshot()
})
