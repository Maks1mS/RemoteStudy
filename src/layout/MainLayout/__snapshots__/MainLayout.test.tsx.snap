// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders 1`] = `
<div>
  <div
    class="css-ihy8jo-Layout e16xpiph0"
  >
    <header
      class="css-3kezaa-Header e16xpiph1"
    >
      <div
        class="css-1xfx2u-MainLayout"
      >
        <a
          class="css-uvbdfj-NavLink-MainLayout e16xpiph3"
        >
          Remote Study
        </a>
      </div>
      <a
        class="css-1ja0cqr-NavLink e16xpiph3"
      >
        Все предметы
      </a>
    </header>
    <main
      class="css-lqeazj-Main e16xpiph2"
    >
      Test
    </main>
  </div>
</div>
`;
